### PR TITLE
NXP-26938: expose variables to configure the create and edit dialog sizes (Backport 9.10)

### PR DESCRIPTION
**Pull request by semisse**

### CODE REVIEW

**Comment by guirenard** [comment]
:+1:

**Comment by semisse** [comment]
Tested locally ok

File: themes/base.html [hunk 1]
```diff
@@ -258,6 +258,12 @@
       min-width: 480px;
     };
 
+    --nuxeo-document-create-popup-height: 100vh;
+    --nuxeo-document-create-popup-width: 100vw;
+
+    --nuxeo-document-form-popup-height: 100vh;
+    --nuxeo-document-form-popup-width: 100vw;
```
Comment (Gabez0r) [inline themes/base.html]: Won't these set the dialogs to fullscreen?

**Comment by guirenard** [comment]
:+1:

**Comment by semisse** [comment]
Since we've changed the approach to this problem, I'm closing this PR.
https://github.com/nuxeo/nuxeo-web-ui/pull/596

**Comment by semisse** [comment]
Since we've changed the approach to this problem, I'm closing this PR.
https://github.com/nuxeo/nuxeo-web-ui/pull/611

**Pull request closed without merging**